### ISSUE TITLE
EDA review

### ISSUE THREAD

**Issue by hermands** [amromeo/api_r2020#23]
@amromeo Love the EDA exercises. Lot's of interesting ideas and teaching points in there.

My thoughts:
- Need library(stringr)
- Mention the significance of "unique" in describing the important columns from database perspective?
-- Is `unique` right here, as this is a de-normalized table?
- I would put the `View()` on its own line, so it's obvious it's there
- explicitly define in intro what is "test_name" and "component", in case not obvious to everyone
- Maybe use `count` or `N` for variable name rather than `n`, so slightly more different from the function `n()`?
- replace "analytical" measurement range with "reportable" measurement range
- had_max_wbc looks right-skewed. I would use wilcox.test rather than t.test
-- maybe take a peak at histograms: 
mimic_sepsis %>%
  ggplot() +
  geom_histogram(aes(x=hadm_max_wbc)) +
  facet_wrap(~sepsis, scales = "free_y")
OR 
look at geom_violin() in addition to box plot

**Comment by amromeo** [comment]
thanks dan, really helpful
doing some reworking here to smoothen out
stringr loads with tidyverse

**Comment by pcmathias** [comment]
@amromeo You may have this lined up but I want to circle back to Dan's feedback about the definitions - particularly making the distinctions about test_name, component, and what charttime actually represents. From what I can tell charttime is associated with the collection time for the sample rather than the result time and answering your challenge question for your turn 3 requires a good understanding.

I found it a little confusing at first that the questions were below the code chunk. I think there is a question of how much we want to encourage true exploration vs. how much we want to present directed goals. If we are emphasizing explicit instruction (even without breaking things down by every step) it might be good to put the questions up front.

Also, I think 5 min might be sufficient for your turn 2.

Overall it looks good and should make for a very interactive session!

**Comment by pcmathias** [comment]
Also, other quick thought: would rephrase challenge for your turn 4 to the middle 95 percent of hemoglobin rather than the reference range since this is not a normal population. I'm being nitpicky but I think that makes it clearer.
